functions/last.mjs

```diff
@@ -0,0 +1,11 @@
+// import curry from './curry';
+import first from './first';
+import reverse from './reverse';
+
+/**
+ * Returns an array containg the last n items of the given array.
+ */
+const last = (xs, n = 1) => reverse(first(reverse(xs), n));
```

Comment:
Haha, I get what you're doing here but I think an implementation that doesn't double-reverse might be easier to understand and probably a bit more performant. 😉 
At least it feels a bit wasteful having to reverse the entire array if you're only interested in the last item.

Reply:
Yep, quickly took it from [this resource](https://medium.com/dailyjs/functional-js-with-es6-recursive-patterns-b7d0813ef9e3) to get some functions in the package, but will write everything from scratch from now on. Should bump quality and improve personal learning 😆 👋